migrate/20210623152412_create_tables.go

```diff
@@ -0,0 +1,366 @@
+package migrate
+
+import (
+	"github.com/go-pg/pg/v10/orm"
+	migrations "github.com/robinjoseph08/go-pg-migrations/v3"
+)
+
+func init() {
+	up := func(db orm.DB) error {
+		_, err := db.Exec(`
+			CREATE TABLE pg_chains (
+                name     TEXT  NOT NULL PRIMARY KEY,
+				tip_hash BYTEA NOT NULL
+			)
+		`)
+		if err != nil {
+			return err
+		}
+
+		_, err = db.Exec(`
+			CREATE TABLE pg_blocks (
+				hash              BYTEA PRIMARY KEY,
+				parent_hash       BYTEA,
+				height            BIGINT NOT NULL,
```

Comment:
Hmm... so it looks like a BIGINT is equivalent to an int64. Which means that we're potentially storing uint64 values in something that's one bit less. This should be fine assuming that we never store math.MaxUint64, but it would break if we did try to store that value. Maybe add a check for it before storing somehow?